tablet-report-parser/tablet-report-parser.pl

```diff
@@ -169,13 +169,22 @@ sub Process_tablet_report{
      SELECT  namespace,table_name, count(*) as total_tablet_count,count(DISTINCT tablet_uuid) as unique_tablet_count, count(DISTINCT node_uuid) as nodes
 	 FROM tablet GROUP BY namespace,table_name;
 CREATE VIEW tablets_per_node AS
-    SELECT node_uuid,min(ip) as node_ip,min(zone) as zone,  count(*) as tablet_count,
+    SELECT node_uuid,ip as node_ip,zone,  count(*) as tablet_count,
```

Comment:
Is `min(ip)` no longer the correct thing to do here? What if there are multiple IPs in this column?

Reply:
min(ip) is incorrect + not required because the tuple (node_uuid, ip) is unique to each node and both are used in the GROUP BY.